src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -147,10 +147,10 @@ This package is an internal implementation of the .NET Core SDK and is not meant
             @(ReferencePathWithRefAssemblies->WithMetadataValue('ReferenceGrouping', 'Microsoft.NETCore.App'));" />
 
       <AspNetCoreReferenceAssemblyPath
-          Include="@(_SelectedExtensionsRefAssemblies->'$(MicrosoftInternalExtensionsRefsPath)%(FileName)%(Extension)')" />
+          Include="@(_SelectedExtensionsRefAssemblies->'$(PkgMicrsoft_Extensions_Internal_Transport)\ref\%(FileName)%(Extension)')" />
 
       <AspNetCoreReferenceDocXml Include="@(_ResolvedProjectReferencePaths->WithMetadataValue('IsReferenceAssembly', 'false')->'%(RootDir)%(Directory)%(FileName).xml')" />
-      <AspNetCoreReferenceDocXml Include="@(_SelectedExtensionsRefAssemblies->'$(MicrosoftInternalExtensionsRefsPath)%(FileName).xml')" />
+      <AspNetCoreReferenceDocXml Include="@(_SelectedExtensionsRefAssemblies->'$(PkgMicrsoft_Extensions_Internal_Transport)\ref\%(FileName).xml')" />
```

Comment:
Todo: the xml files are actually located in lib\

Reply:
Should be resolved now.